lib/usagereporter/teleport/aggregating/service.go

```diff
@@ -35,8 +36,9 @@ const (
 )
 
 const (
-	userActivityReportsPrefix = "userActivityReports"
-	userActivityReportsLock   = "userActivityReportsLock"
+	userActivityReportsPrefix     = "userActivityReports"
+	userActivityReportsLock       = "userActivityReportsLock"
+	ResourcePresenceReportsPrefix = "ResourcePresenceReports"
```

Comment:
Backend prefixes are all camelCase, not PascalCase, we should stick to the existing style.
```suggestion
	ResourcePresenceReportsPrefix = "resourcePresenceReports"
```